sessions/css-layout-with-flexbox/personal-travel-diary/css/styles.css

```diff
@@ -0,0 +1,60 @@
+* {
+	box-sizing: border-box;
+}
+
+/* Exercise 1: Adjust the overall layout with Flexbox according to the following criteria:
+
+- The content is aligned in the center and should stay in the center when changing the size of the viewport.
+- The heading "My Personal Travel Diary" should also be aligned in the center of the page.
+- Note: The layout of the article elements should stay the same!
+
+*/
+
+body {
+	line-height: 1.5;
+	font-family: sans-serif;
+}
+
+/* end of exercise 1 - scroll down to exercise 2 */
+
+h1,
+h2 {
+	line-height: 1.2;
+	border-bottom: 5px solid steelblue;
+	text-align: center;
+	display: inline-block;
```

Comment:
I'm not quite sure if this is necessary and/or good practice.

Reply:
The `inline-block` is required to have the underline below the text as long as the content